### PR TITLE
[WIP] Updating scalability doc to include info about new distributor config

### PR DESCRIPTION
**Pull request by mat-mcloughlin**
Just a small note about the distributor being retired. 

Relates to https://github.com/Particular/docs.particular.net/pull/1155

### CODE REVIEW

File: nservicebus/scalability-and-ha/distributor/index.md [hunk 1]
```diff
@@ -192,6 +192,7 @@ Similar to standard NServiceBus routing, you do not want high priority messages
 
 In this case, name the queues just like the messages. For example, `SubmitPurchaseOrder.StrategicCustomers.Sales`. This is the name of the distributor's data queue and the input queues of each of the workers.
 
+NOTE: In version 6 We have retired the distributor and you should call `EndpointConfiguration.EnlistWithLegacyMSMQDistributor(masterNodeAddress, masterNodeControlAddres, capacity)` to enlist with a legacy distributor.
```
Comment (SimonCropp) [inline nservicebus/scalability-and-ha/distributor/index.md]: Can u make this a snippet so it is a little more resiliant to refactorings

Comment (SimonCropp) [inline nservicebus/scalability-and-ha/distributor/index.md]: remove the personal voice of we and you

**Comment by SimonCropp** [comment]
@mat-mcloughlin still working on this or do you want to hand it over to someone else?

**Comment by mat-mcloughlin** [comment]
Sorry @SimonCropp I got distracted with some other work. It'll be done today

**Comment by mat-mcloughlin** [comment]
What was a simple a one line change is turning into a bit more as there is no v6 distributor. Need to speak to @janovesk about this first

**Comment by SimonCropp** [comment]
@mat-mcloughlin ok i will mark this as WIP for now. remove the wip when u have clarified the above.

**Comment by SimonCropp** [comment]
removing @mikeminutillo as assigned since he does not have the context to move this forward

**Comment by weralabaj** [comment]
This change was covered in https://github.com/Particular/docs.particular.net/pull/1326.

**Pull request closed without merging**